03-Quotes (copy)/quotes.js

```diff
@@ -9,27 +25,23 @@
 // }
 
 
-// function validateQuotes(){
-//   if(limit.value.length<=0){
-//     alert("Enter Limit")
-//     limit.focus()
-//     return false
-// }
 
 function validateAuthor(){
   let limit=document.querySelector('#limit')
   let name=document.querySelector('#name')
  if(limit.value.length<=0){
```

Comment:
if all your fields are empty there should be only one check instead of giving multiple alert popups to the user